sdk/eventhub/Microsoft.Azure.Management.EventHub/src/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides developers with a library to create and manage all Azure Event Hubs resources.")]
 
 [assembly: AssemblyVersion("2.0.0.0")]
```

Comment:
Please also bump the `AssemblyVersion` here.

Reply:
@Yao725 , when do we upgrade assembly version and assembly file version ? is it only during major bump up.

Reply:
Our suggestion is that every time you update the SDK, you should also update these two versions here. You can see more details about how to use these two versions [here](https://docs.microsoft.com/en-us/troubleshoot/developer/visualstudio/general/assembly-version-assembly-file-version).